proto/info/info.proto

```diff
@@ -912,6 +912,17 @@ message ZInfoSnapshot {
   ErrorInfo snap_err = 6;
 }
 
+enum ZInfoClusterNodeStatus {
+  Node_Status_INVALID = 0;
+  Node_Status_READY = 1;    // cluster reports our node is ready
+  Node_Status_NOTREADY = 2; // cluster reports our node is ready
```

Comment:
Comments are identical for the two fields.

But how does the state of the node relate to the state of the cluster?
Take the case of a cluster where its config lists node1, node2, and node3.
When e.g., node3 reports READY does it mean that not only it has joined the cluster but also that it sees node2 and node1 as part of the cluster?

What status do we expect to see reported by the nodes if/when one of the nodes fail to join the cluster? Will the other two report it as READY?

These semantics should presumably be described in comments in the proto file.

Reply:
I think this is only for the node to report the status as part of the cluster. We'll need another info or metric message to report the cluster status (more like kubernetes view) w/ an elected reporter on the cluster. But that is more complex and is not part of this PR.
So, this is only saying as a device, my node in this cluster status. If all nodes report 'unknown' then the cluster is down.

Reply:
Can you 1) correct the comment and 2) add the above information as comments.

Also, does it make sense to have a separate value for DOWN vs. using UNKNOWN? Either one works for me as long as this is clearly documented.